packages/cli/package-lock.json

```diff
@@ -4,4295 +4,4543 @@
 	"lockfileVersion": 1,
```

Comment:
this file would normally be committed, but since there are no dependency changes here, there shouldn't be any changes as well.
it's likely a leftover of the one time `npm link` on the cli package to symlink `markbind`, you can safely discard this.

I'll see to adding a note after that step in the devguide sometime